### PR TITLE
Update to 1.9.3, using packaged build

### PR DESCRIPTION
**Pull request by gasinvein**
Some deleted patches _may_ be needed, but at least on x86_64 Telegram seems to build and work file.

### CODE REVIEW

**Comment by gasinvein** [comment]
@grulja Can you please merge #120 and change base branch here to `beta`? Since patches here are not yet accepted upstream and are subject to further changes, I consider this kind of beta build.

**Comment by grulja** [comment]
Looks that all the patches can be dropped, except the one for fontconfig? Can you add it again? You can take rebased version from my PR.

**Comment by gasinvein** [comment]
@grulja Done. It moved to `lib_ui` submodule.

File: org.telegram.desktop.json [hunk 1]
```diff
@@ -98,91 +87,173 @@
                 }
             ]
         },
+        {
+            "name": "cmake",
```
Comment (grulja) [inline org.telegram.desktop.json]: Why do you build your own CMake? It's definitely in the runtime.

File: org.telegram.desktop.json [hunk 2]
```diff
@@ -98,91 +87,173 @@
...
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/AbiWord/enchant/releases/download/v2.2.7/enchant-2.2.7.tar.gz",
+                    "sha256": "1b22976135812b35cb5b8d21a53ad11d5e7c1426c93f51e7a314a2a42cab3a09"
+                }
+            ]
+        },
+        {
+            "name": "lz4",
```
Comment (grulja) [inline org.telegram.desktop.json]: lz4 seems to also be in the runtime

File: org.telegram.desktop.json (hunk 1 above)
Comment (gasinvein) [reply org.telegram.desktop.json]: Telegram wants cmake 3.16, while in the runtime there is 3.15.x

File: org.telegram.desktop.json (hunk 2 above)
Comment (gasinvein) [reply org.telegram.desktop.json]: Nope, no lz4 in the runtime.

File: org.telegram.desktop.json (hunk 1 above)
Comment (grulja) [reply org.telegram.desktop.json]: Is there a particular reason why this version is needed? If they don't use any specific feature we can patch it and use cmake from the runtime.

File: org.telegram.desktop.json (hunk 2 above)
Comment (grulja) [reply org.telegram.desktop.json]: Ok, you seem to be right. I was checking source code of the freedesktop runtime and saw it so I assumed it's there.

File: org.telegram.desktop.json (hunk 1 above)
Comment (gasinvein) [reply org.telegram.desktop.json]: I don't know, probably? Anyway, I believe building own cmake module is way easier than maintaining yet another patch.

**Comment by gasinvein** [comment]
@grulja RPMFusion folks are [removing](https://github.com/EasyCoding/tgbuild/blob/3fcdb2f183c583e4526fd69d37ccc35daac55c85/telegram-desktop-build-fixes.patch#L44-L46) `fc-custom.conf` from resources altogether, so that Telegram uses system font config. Maybe we should do the same instead of adding flatpak-specific font paths?

**Comment by grulja** [comment]
> @grulja RPMFusion folks are [removing](https://github.com/EasyCoding/tgbuild/blob/3fcdb2f183c583e4526fd69d37ccc35daac55c85/telegram-desktop-build-fixes.patch#L44-L46) `fc-custom.conf` from resources altogether, so that Telegram uses system font config. Maybe we should do the same instead of adding flatpak-specific font paths?

Ok. Go ahead.

**Pull request merged**